src/notifications/notifications.service.ts

```
@@ -204,20 +204,21 @@ export class NotificationsService {
     }
   }
 
-  async markAsRead(notificationId: string, userId: string) {
-    const notification = await this.notificationRepository.findOne({
-      where: { id: notificationId, userId },
-    });
+async markAsUnread(notificationId: string, userId: string) {
```

ESLint

[error] Insert `··`

(prettier/prettier)